exercises/concept/freelancer-rates/.meta/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "authors": [
+    "vaeng"
+  ],
+  "files": {
+    "solution": [
+      "freelancer_rates.cpp"
+    ],
+    "test": [
+      "freelancer_rates_test.cpp"
+    ],
+    "exemplar": [
+      ".meta/exemplar.cpp"
+    ]
+  },
+  "icon": "lasagna",
```

Comment:
I'm assuming this is a copy-paste of the lasagna exercise - does Exercism have a good icon for this exercise already?